test/common/test_eigen.cpp

```diff
@@ -932,6 +932,13 @@ TEST (PCL, transformPlane)
   transformationd.linear() = (Eigen::Matrix3d) Eigen::AngleAxisd(M_PI/7, Eigen::Vector3d::UnitY())
   * Eigen::AngleAxisd(M_PI/4, Eigen::Vector3d::UnitZ());
   test << 5.35315, 2.89914, 0.196848, -49.2788;
+
+  // From Eigen 3.3.2 onwards, the Eigen::Hyperplane::transform always returns
+  // coefficients with the normal normalized
+  #if EIGEN_VERSION_AT_LEAST (3, 3, 2)
+    test /= test.head<3> ().norm ();
```

Comment:
Shouldn't we rather tweak the function to return consistent results no matter what the Eigen version is? That is, to return normalized normal?